app-misc/asciinema/asciinema-2.0.2.recipe

```diff
@@ -46,11 +45,9 @@ BUILD()
 
 INSTALL()
 {
-	sed -i \
-		-e "s|documentation/man|$relativeManDir|" \
-		-e "s|documentation/packages/asciinema|$relativeDocDir|" \
-		setup.py
-	python3 setup.py install --prefix=$prefix --root=/ --skip-build
+	sed -i "s|share/doc/asciinema|$relativeDocDir|
+		s|share/man|$relativeManDir|" setup.py
```

Comment:
I think this should go into `PATCH()` instead.

Reply:
I placed it there because of [this conversation](https://github.com/haikuports/haikuports/pull/3219#discussion_r225128937), which *seems* to apply here. Relevant quote:

> Move the sed commands from PATCH to INSTALL. That's because, imho, the changes are not a real patch but a requirement for the installation.

He might have meant that because they were *meant* to be replaced, though, but he also let this pass when he reviewed it the previous version of this recipe, as well as in another I submitted. Maybe we should ask him to be sure?

Reply:
This `sed` is applied directly into the source, meaning that it will work once and only once, so it shouldn't be in `INSTALL()`. Also, `PATCH()` is un-upstreamable by design as factors like cross-platform support can't usually be expressed there.

We can ask @fbrosson of course, but he's MIA atm. If you have a good idea on why something shouldn't be like that, you can always convince him :)

Reply:
My bad; just disregard what I said earlier, please. I was confused.